README.md

```diff
@@ -4,9 +4,9 @@ redux
 [![build status](https://img.shields.io/travis/gaearon/redux.svg?style=flat-square)](https://travis-ci.org/gaearon/redux)
 [![npm version](https://img.shields.io/npm/v/redux.svg?style=flat-square)](https://www.npmjs.com/package/redux)
 
-Atomic Flux with hot reloading.  
+Atomic Flux with hot reloading.
 
-**The API is likely to change a few times before we reach 1.0.**  
```

Comment:
I know, I did the same "mistake" ;)

It's just that @gaearon wants to have it like this, so we should follow his conventions.